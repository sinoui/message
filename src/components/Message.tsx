import styled from 'styled-components';

const Message = styled.div`
  display: inline-flex;
  align-items: flex-start;
  padding: 10px 16px;
  margin: 4px 0;
  background-color: ${(props) => props.theme.palette.background.paper};
  box-shadow: ${(props) => props.theme.shadows[1]};
  border-radius: 4px;
  pointer-events: all;
`;

export default Message;
