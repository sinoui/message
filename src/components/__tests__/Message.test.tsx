import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import TestWrapper from './TestWrapper';
import Message from '../Message';

describe('Message 镜像测试', () => {
  it('渲染Message', () => {
    const tree = renderer
      .create(
        <TestWrapper>
          <Message />
        </TestWrapper>,
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
