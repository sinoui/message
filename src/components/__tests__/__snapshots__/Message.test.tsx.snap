// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message 镜像测试 渲染Message 1`] = `
.c0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  padding: 10px 16px;
  margin: 4px 0;
  background-color: #ffffff;
  box-shadow: 0px 1px 3px 0px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 2px 1px -1px rgba(0,0,0,0.12);
  border-radius: 4px;
  pointer-events: all;
}

<div
  className="c0"
/>
`;
